src/main/java/uk/co/onsdigital/discovery/metadata/api/controller/MetadataController.java

```diff
@@ -99,14 +111,6 @@ public ErrorResponse onUnsupportedOperation(final UnsupportedOperationException
         return new ErrorResponse(HttpStatus.NOT_IMPLEMENTED, ex.getMessage());
     }
 
-    @ResponseBody
```

Comment:
This was a mistake committed in a previous ticket - it overrides all the other handlers, so everything gets reported as a 500 error.